hazelcast-client/src/main/java/com/hazelcast/client/impl/ClientDynamicClusterConfig.java

```diff
@@ -402,7 +416,9 @@ public Config addFlakeIdGeneratorConfig(FlakeIdGeneratorConfig flakeIdGeneratorC
                 flakeIdGeneratorConfig.getName(),
                 flakeIdGeneratorConfig.getPrefetchCount(),
                 flakeIdGeneratorConfig.getPrefetchValidityMillis(),
-                flakeIdGeneratorConfig.getIdOffset());
+                flakeIdGeneratorConfig.getIdOffset(),
+                // TODO this should be flakeIdGeneratorConfig.isStatisticsEnabled()
+                true);
```

Comment:
Client protocol version 1.6.0-8 includes changes from https://github.com/hazelcast/hazelcast-client-protocol/pull/117 and https://github.com/hazelcast/hazelcast-client-protocol/pull/115. The Hazelcast-side PRs are this one and #12317. `FlakeIdGeneratorConfig.statisticsEnabled` will be introduced by #12317; in order for this PR to compile and turn green, I added a hard-coded value here instead of `flakeIdGeneratorConfig.isStatisticsEnabled()`. This should be replaced by the property in #12317.